sphinx_needs/layout.py

```diff
@@ -456,7 +455,28 @@ def meta(self, name: str, prefix: Optional[str] = None, show_empty: bool = False
         elif data is None and show_empty:
             data = ""
 
-        if isinstance(data, str):
+        try:
+            ismultiline = self.need["attribute_metadata"][name]["multiline"]
+        except:
+            ismultiline = False
+
+        try:
+            isuml = self.need["attribute_metadata"][name]["uml"]
+        except:
+            isuml = False
+
+        if isuml:
```

Comment:
uml part has to be shifted from needuml. It would be nice to refer to uml with <need id>.attibute.

Reply:
I implemented a protoype to provide an exmaple requested [here](https://github.com/useblocks/sphinxcontrib-needs/issues/613#issuecomment-1160583656)